pcsx2/x86/microVU_IR.h

```diff
@@ -330,6 +330,16 @@ class microRegAlloc
 		}
 	}
 
+	bool checkVFClamp(int regId)
+	{
+		if(CHECK_SKIPMPEGHACK)
```

Comment:
Formatting.

Reply:
Are you freaking serious?

Reply:
Space after the if.

Reply:
It's temporary code... do you really think I'm leaving a skipmpeg link in the VU code? Use your brain LT

Reply:
Okay, didn't know it was temporary, ignore the comment then.